lib/core/config.rb

```diff
@@ -71,6 +76,57 @@ def ensure_config_app!(app_name, app_options)
     raise "App '#{app_name}' is empty in 'controlplane.yml'." unless app_options
   end
 
+  def ensure_org!
+    return if @org
+
+    raise "No org provided. The org can be provided either through --org " \
+          "or the CPLN_ORG env var ('allow_org_override_by_env' must be set to true in 'controlplane.yml')."
+  end
+
+  def ensure_app!
+    return if @app
+
+    raise "No app provided. The app can be provided either through --app " \
+          "or the CPLN_APP env var ('allow_app_override_by_env' must be set to true in 'controlplane.yml')."
+  end
+
+  def ensure_required_options!
+    missing_str = required_options
+                  .reject { |option_name| %i[org app].include?(option_name) || options.key?(option_name) }
+                  .map { |option_name| "--#{option_name}" }
+                  .join(", ")
+
+    raise "Required options missing: #{missing_str}" unless missing_str.empty?
+  end
+
+  def non_empty_str_or_nil(str)
```

Comment:
Should be static. Might be in a helpers class. Or here is fine, b/c this config is where we process? but if we need both for cmd args and yml, then maybe a help class?